.github/workflows/ci.yml

```diff
@@ -5,11 +5,13 @@ on:
     branches:
       - main
       - '*x'
+      - stcal-alignment
     tags:
       - '*'
   pull_request:
     branches:
       - main
+      - stcal-alignment
```

Comment:
This and line 8 will need to be removed before merging.

Reply:
Removed.